README.md

```diff
@@ -81,3 +81,212 @@ The following options are specific to this plugin:
   include A/B testing or localization specific output. Note: if you
   invoke the callback more than once, you should change the output
   filename to avoid overwriting previous invocations' output.
+
+
+## Examples
```

Comment:
consider splitting this out into a separate file e.g. `EXAMPLES.md` or move it to `examples/README.md`

you can then link to it from this README